tests/sync/test_rest_api.py

```
@@ -238,6 +238,7 @@ async def test_denial_end_to_end(self):
             pass
         finally:
             await seb_communicator.disconnect()
+        await asyncio.sleep(5) # Give a second for the fire and forget pubmed to run
```

Ruff (0.8.2)

Undefined name `asyncio`

(F821)